src/dataViews/DataViewExpander.ts

```diff
@@ -1266,11 +1266,33 @@ export class DataViewExpander {
         const matchedAttributeDTOResult = await this.consumption.attributes.executeIQLQuery({ query });
         const matchedAttributeDVOs = await this.expandLocalAttributeDTOs(matchedAttributeDTOResult.value);
 
+        let valueType;
+        let renderHints;
+        let valueHints;
+
+        // Check if we've the same valueType throughout all matched attributes, if yes, set valueType and renderHints.
+        // If not, delete valueType and renderHints
+        for (const item of matchedAttributeDVOs) {
+            if (!valueType) {
+                valueType = item.valueType;
+                renderHints = item.renderHints;
+                valueHints = item.valueHints;
+            }
+            if (valueType === item.valueType) continue;
+            valueType = undefined;
+            renderHints = undefined;
+            valueHints = undefined;
+            break;
+        }
```

Comment:
```suggestion
        const { valueType, renderHints, valueHints } = matchedAttributeDVOs.every((dvo) => dvo.valueType === matchedAttributeDVOs[0].valueType)
        ? { valueType: matchedAttributeDVOs[0].valueType, renderHints: matchedAttributeDVOs[0].renderHints, valueHints: matchedAttributeDVOs[0].valueHints }
        : { valueType: undefined, renderHints: undefined, valueHints: undefined };
```
what about this? would make a bit clearer that every valueType is compared.